# Korean translation for apport
# Copyright (c) (c) 2006 Canonical Ltd, and Rosetta Contributors 2006
# This file is distributed under the same license as the apport package.
# Eungkyu Song <eungkyu@gmail.com>, 2006.
#
msgid ""
msgstr ""
"Project-Id-Version: apport\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2022-12-07 16:35+0100\n"
"PO-Revision-Date: 2021-01-23 02:37+0000\n"
"Last-Translator: Catry <Unknown>\n"
"Language-Team: Korean <ko@li.org>\n"
"Language: ko\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2022-12-07 14:36+0000\n"
"X-Generator: Launchpad (build 41e7553f8097ce3a683f90e835708cbe7bf0006c)\n"

#: ../apport/com.ubuntu.apport.policy.in.h:1
msgid "Collect system information"
msgstr "시스템 정보 수집"

#: ../apport/com.ubuntu.apport.policy.in.h:2
msgid ""
"Authentication is required to collect system information for this problem "
"report"
msgstr "이 문제에 대한 시스템 정보를 수집하려면 인증이 필요합니다."

#: ../apport/com.ubuntu.apport.policy.in.h:3
msgid "System problem reports"
msgstr "시스템 문제 보고"

#: ../apport/com.ubuntu.apport.policy.in.h:4
msgid "Please enter your password to access problem reports of system programs"
msgstr "시스템 프로그램의 문제 보고서에 접근하려면 암호를 입력해야 합니다."

#: ../apport/ui.py:165
msgid "This package does not seem to be installed correctly"
msgstr "이 패키지를 올바르게 설치하지 않았습니다."

#: ../apport/ui.py:175
#, python-format
msgid ""
"This does not seem to be an official %s package. Please retry after updating "
"the indexes of available packages, if that does not work then remove related "
"third party packages and try again."
msgstr ""
"공식 %s 패키지가 아닌 것 같습니다. 사용 가능한 패키지의 인덱스를 업데이트 한 "
"후 다시 시도하시오. 그래도 작동하지 않으면 관련 타사 패키지를 제거하고 다시 "
"시도하시오."

#: ../apport/ui.py:205
#, python-format
msgid ""
"You have some obsolete package versions installed. Please upgrade the "
"following packages and check if the problem still occurs:\n"
"\n"
"%s"
msgstr ""
"오래된 버전의 패키지가 몇 개 설치되어 있습니다. 다음 패키지를 업그레이드한 "
"후 계속 문제가 발생하는지 점검해 주십시오:\n"
"\n"
"%s"

#: ../apport/ui.py:359
msgid "unknown program"
msgstr "알 수 없는 문제"

#: ../apport/ui.py:362
#, python-format
msgid "Sorry, the program \"%s\" closed unexpectedly"
msgstr "죄송합니다. \"%s\" 프로그램이 예상치 않게 끝났습니다."

#: ../apport/ui.py:365 ../apport/ui.py:1899
#, python-format
msgid "Problem in %s"
msgstr "%s 내의 문제"

#: ../apport/ui.py:370
msgid ""
"Your computer does not have enough free memory to automatically analyze the "
"problem and send a report to the developers."
msgstr ""
"컴퓨터에 자동으로 문제를 분석하여 개발자에게 보고서를 보낼 수 있을 정도의 여"
"유 메모리가 없습니다."

#: ../apport/ui.py:430 ../apport/ui.py:448 ../apport/ui.py:621
#: ../apport/ui.py:630 ../apport/ui.py:867 ../apport/ui.py:1677
#: ../apport/ui.py:1860 ../apport/ui.py:1865
msgid "Invalid problem report"
msgstr "올바르지 않은 문제 보고서"

#: ../apport/ui.py:431
msgid "You are not allowed to access this problem report."
msgstr "이 문제 보고서에 접근할 수 있는 권한이 없습니다."

#: ../apport/ui.py:439
msgid "Error"
msgstr "오류"

#: ../apport/ui.py:441
msgid "There is not enough disk space available to process this report."
msgstr "디스크 공간이 부족하여 보고서를 처리할 수 없습니다."

#: ../apport/ui.py:479
msgid "No PID specified"
msgstr "PID가 지정되지 않음"

#: ../apport/ui.py:481
msgid "You need to specify a PID. See --help for more information."
msgstr "PID를 지정해야 합니다. 더 많은 정보는 --help를 통해 확인하시오."

#: ../apport/ui.py:492 ../apport/ui.py:597
msgid "Invalid PID"
msgstr "유효하지 않은 PID"

#: ../apport/ui.py:493
msgid "The specified process ID does not exist."
msgstr "지정된 프로세스 ID가 존재하지 않습니다."

#: ../apport/ui.py:498
msgid "Not your PID"
msgstr "당신의 PID가 아님"

#: ../apport/ui.py:499
msgid "The specified process ID does not belong to you."
msgstr "지정된 프로세스 ID는 당신의 소유가 아닙니다."

#: ../apport/ui.py:554
msgid "No package specified"
msgstr "패키지가 지정되지 않음"

#: ../apport/ui.py:556
msgid ""
"You need to specify a package or a PID. See --help for more information."
msgstr ""
"패키지 혹은 PID를 지정해야 합니다. --help 옵션으로 더 많은 정보를 확인할 수 "
"있습니다."

#: ../apport/ui.py:584
msgid "Permission denied"
msgstr "권한 없음"

#: ../apport/ui.py:586
msgid ""
"The specified process does not belong to you. Please run this program as the "
"process owner or as root."
msgstr ""
"선택한 프로세스는 여러분이 실행한 것이 아닙니다. 이 프로그램을 직접 실행하거"
"나 root 사용자로 실행하십시오."

#: ../apport/ui.py:599
msgid "The specified process ID does not belong to a program."
msgstr "확인된 프로세스 ID는 프로그램에 종속되지 않은 것입니다."

#: ../apport/ui.py:623
#, python-format
msgid "Symptom script %s did not determine an affected package"
msgstr "증상 스크립트 %s에서 영향을 받는 패키지를 확인하지 않았습니다."

#: ../apport/ui.py:631
#, python-format
msgid "Package %s does not exist"
msgstr "%s 패키지가 존재하지 않습니다"

#: ../apport/ui.py:660 ../apport/ui.py:872 ../apport/ui.py:905
#: ../apport/ui.py:915
msgid "Cannot create report"
msgstr "보고서를 만들 수 없습니다"

#: ../apport/ui.py:675 ../apport/ui.py:734 ../apport/ui.py:752
msgid "Updating problem report"
msgstr "문제 보고서 갱신"

#: ../apport/ui.py:677
msgid ""
"You are not the reporter or subscriber of this problem report, or the report "
"is a duplicate or already closed.\n"
"\n"
"Please create a new report using \"apport-bug\"."
msgstr ""
"이 문제를 보고한 사람이 아니거나 문제 보고서를 구독하지 않거나 중복된 보고서"
"거나 이미 닫힌 문제입니다.\n"
"\"apport-bug\" 명령을 이용해 새 보고서를 만들어주십시오."

#: ../apport/ui.py:689
msgid ""
"You are not the reporter of this problem report. It is much easier to mark a "
"bug as a duplicate of another than to move your comments and attachments to "
"a new bug.\n"
"\n"
"Subsequently, we recommend that you file a new bug report using \"apport-"
"bug\" and make a comment in this bug about the one you file.\n"
"\n"
"Do you really want to proceed?"
msgstr ""
"이 문제 보고서를 만든 사람이 아닙니다. 다른 버그와 중복되는 버그로 표시하고 "
"코멘트와 첨부 파일을 새 버그로 옮기는 것이 훨씬 간단합니다.\n"
"따라서 \"apport-bug\" 명령을 이용해 새 버그를 보고한 후 보고한 버그 보고서에 "
"코멘트를 작성할 것을 권장합니다.\n"
"정말로 이대로 진행하겠습니까?"

#: ../apport/ui.py:735 ../apport/ui.py:753
msgid "No additional information collected."
msgstr "어떠한 추가 정보도 수집하지 않습니다."

#: ../apport/ui.py:816
msgid "What kind of problem do you want to report?"
msgstr "어떤 종류의 문제를 보고하시겠습니까?"

#: ../apport/ui.py:837
msgid "Unknown symptom"
msgstr "알려지지 않은 증상"

#: ../apport/ui.py:838
#, python-format
msgid "The symptom \"%s\" is not known."
msgstr "\"%s\"은(는) 알려지지 않은 증상입니다."

#: ../apport/ui.py:874
msgid ""
"The window option cannot be used on Wayland.\n"
"\n"
"Please find the window's process ID and then run 'ubuntu-bug <process ID>'.\n"
"\n"
"The process ID can be found by running the System Monitor application. In "
"the Processes tab, scroll until you find the correct application. The "
"process ID is the number listed in the ID column."
msgstr ""

#: ../apport/ui.py:890
msgid ""
"After closing this message please click on an application window to report a "
"problem about it."
msgstr "이 메시지를 닫은 후 문제를 보고할 프로그램의 창을 선택해주십시오."

#: ../apport/ui.py:907 ../apport/ui.py:916
msgid "xprop failed to determine process ID of the window"
msgstr "xprop 프로그램이 창 프로세스 ID를 알아낼 수 없습니다."

#: ../apport/ui.py:931
#, python-format
msgid "%(prog)s <report number>"
msgstr ""

#: ../apport/ui.py:932
msgid "Specify package name."
msgstr "패키지 이름을 지정합니다."

#: ../apport/ui.py:938 ../apport/ui.py:1070
msgid "Add an extra tag to the report. Can be specified multiple times."
msgstr "보고서에 태그를 추가합니다. 여러 번 지정할 수 있습니다."

#: ../apport/ui.py:976
#, python-format
msgid ""
"%(prog)s [options] [symptom|pid|package|program path|.apport/.crash file]"
msgstr ""

#: ../apport/ui.py:987
msgid ""
"Start in bug filing mode. Requires --package and an optional --pid, or just "
"a --pid. If neither is given, display a list of known symptoms. (Implied if "
"a single argument is given.)"
msgstr ""
"버그 보고 모드로 시작. 인수로는 --package와 --pid 혹은 --pid 옵션을 단독으로 "
"사용합니다. 어떤 것도 사용하지 않으면 알려진 증상의 목록을 표시합다. (하나의 "
"인수가 주어진 PID가 유일한 인수일 경우 자동으로 실행합니다.)"

#: ../apport/ui.py:996
msgid "Click a window as a target for filing a problem report."
msgstr "문제를 보고할 프로그램의 창을 선택해주십시오."

#: ../apport/ui.py:1005
msgid "Start in bug updating mode. Can take an optional --package."
msgstr ""
"버그 업데이트 모드로 시작. 추가적으로 --package 명령을 사용할 수 있습니다."

#: ../apport/ui.py:1014
msgid ""
"File a bug report about a symptom. (Implied if symptom name is given as only "
"argument.)"
msgstr ""
"문제 증상에 대한 버그 보고. (문제 증상만이 매개 변수로 주어진 경우 이 모드가 "
"기본입니다)"

#: ../apport/ui.py:1023
msgid ""
"Specify package name in --file-bug mode. This is optional if a --pid is "
"specified. (Implied if package name is given as only argument.)"
msgstr ""
"--file-bug 모드에서 패키지 이름을 지정합니다. --pid 명령을 지정하면 사용하지 "
"않을 수 있습니다.(패키지 이름이 유일한 인수일 경우 자동으로 실행합니다.)"

#: ../apport/ui.py:1034
msgid ""
"Specify a running program in --file-bug mode. If this is specified, the bug "
"report will contain more information.  (Implied if pid is given as only "
"argument.)"
msgstr ""
"--file-bug 모드로 실행하고 있는 프로그램을 지정해주십시오. 이 항목을 지정하"
"면 버그 보고에 더 많은 정보를 포함하게 됩니다. (PID가 유일한 인수일 경우 자동"
"으로 실행합니다.)"

#: ../apport/ui.py:1042
msgid "The provided pid is a hanging application."
msgstr "주어진 PID는 응답이 없는 프로그램입니다."

#: ../apport/ui.py:1050
#, python-format
msgid ""
"Report the crash from given .apport or .crash file instead of the pending "
"ones in %s. (Implied if file is given as only argument.)"
msgstr ""
"%s에서 대기 중인 것 대신 주어진 .apport 혹은 .crash 파일의 충돌을 보고.(유일"
"한 인수일 경우 자동으로 실행합니다.)"

#: ../apport/ui.py:1060
msgid ""
"In bug filing mode, save the collected information into a file instead of "
"reporting it. This file can then be reported later on from a different "
"machine."
msgstr ""
"버그 정리 모드에서 문제를 보고하지 않고 수집한 정보를 파일로 저장합니다. 이 "
"파일을 이용해 이후 다른 컴퓨터에서 문제를 보고할 수 있습니다."

#: ../apport/ui.py:1078
msgid "Print the Apport version number."
msgstr "Apport 버전 번호를 출력합니다."

#: ../apport/ui.py:1240
msgid ""
"This will launch apport-retrace in a terminal window to examine the crash."
msgstr "충돌을 분석하기 위해 터미널 창에서 apport-retrace 명령을 실행합니다."

#: ../apport/ui.py:1244
msgid "Run gdb session"
msgstr "GDB 세션 실행"

#: ../apport/ui.py:1245
msgid "Run gdb session without downloading debug symbols"
msgstr "디버그 심볼을 다운로드하지 않고 GDB 세션을 실행합니다."

#. TRANSLATORS: %s contains the crash report file name
#: ../apport/ui.py:1247
#, python-format
msgid "Update %s with fully symbolic stack trace"
msgstr "%s을(를) 심볼릭 스택 트레이스로 업데이트합니다."

#: ../apport/ui.py:1309
msgid "Can't remember send report status settings"
msgstr "보내는 리포트 상태 설정을 기억할 수 없습니다."

#: ../apport/ui.py:1313
msgid ""
"Saving crash reporting state failed. Can't set auto or never reporting mode."
msgstr ""
"충돌 보고 상태를 저장하지 못했습니다. 보고 모드를 자동으로 설정할 수 없거나 "
"아예 설정할 수 없습니다."

#: ../apport/ui.py:1396 ../apport/ui.py:1409
msgid ""
"This problem report applies to a program which is not installed any more."
msgstr "이 문제 보고서는 아직 설치하지 않은 프로그램에도 적용됩니다."

#: ../apport/ui.py:1433
#, python-format
msgid ""
"The problem happened with the program %s which changed since the crash "
"occurred."
msgstr "충돌 발생 후 바뀐 프로그램 %s에서 문제가 발생했습니다."

#: ../apport/ui.py:1494 ../apport/ui.py:1618 ../apport/ui.py:1869
msgid "This problem report is damaged and cannot be processed."
msgstr "문제 보고서가 손상되어 처리할 수 없습니다."

#: ../apport/ui.py:1503
msgid "This report is about a package that is not installed."
msgstr "이 보고서는 설치되지 않은 패키지에 대한 것입니다."

#: ../apport/ui.py:1511
msgid "An error occurred while attempting to process this problem report:"
msgstr "이 문제를 보고하는 과정에 문제가 발생했습니다:"

#: ../apport/ui.py:1528
msgid ""
"You have two versions of this application installed, which one do you want "
"to report a bug against?"
msgstr ""
"해당 프로그램은 두 가지 버전으로 설치되었습니다, 어느 것에 대한 버그를 보고하"
"시겠습니까?"

#: ../apport/ui.py:1533
#, python-format
msgid "%s snap"
msgstr "%s 스냅"

#: ../apport/ui.py:1534
#, python-format
msgid "%s deb package"
msgstr "%s deb 패키지"

#: ../apport/ui.py:1575
#, python-format
msgid "%s is provided by a snap published by %s. Contact them via %s for help."
msgstr ""
"%s 은(는) %s  배포한 스냅에서 제공합니다. 도움이 필요하면 %s 연락하십시오."

#: ../apport/ui.py:1580
#, python-format
msgid ""
"%s is provided by a snap published by %s. No contact address has been "
"provided; visit the forum at https://forum.snapcraft.io/ for help."
msgstr ""
"%s 은(는) %s 배포한 스냅에서 제공합니다. 연락처가 제공되지 않았습니다. 도움"
"이 필요하면 https://forum.snapcraft.io/ 포럼을 방문하십시오."

#: ../apport/ui.py:1679
msgid "Could not determine the package or source package name."
msgstr "패키지나 소스 패키지 이름을 알아낼 수 없습니다."

#: ../apport/ui.py:1704
msgid "Unable to start web browser"
msgstr "웹 브라우저를 시작할 수 없습니다"

#: ../apport/ui.py:1705
#, python-format
msgid "Unable to start web browser to open %s."
msgstr "웹 브라우저를 이용하여 %s을(를) 열 수 없습니다."

#: ../apport/ui.py:1815
msgid "Network problem"
msgstr "네트워크 문제"

#: ../apport/ui.py:1819
msgid ""
"Cannot connect to crash database, please check your Internet connection."
msgstr "충돌한 데이터베이스에 연결할 수 없습니다. 인터넷 연결을 확인하세요."

#: ../apport/ui.py:1851
msgid "Memory exhaustion"
msgstr "메모리 부족"

#: ../apport/ui.py:1853
msgid "Your system does not have enough memory to process this crash report."
msgstr "시스템에 충돌 보고서를 처리할 수 있을 만큼의 여유 메모리가 없습니다."

#: ../apport/ui.py:1904
#, python-format
msgid ""
"The problem cannot be reported:\n"
"\n"
"%s"
msgstr ""
"문제를 보고할 수 없습니다:\n"
"\n"
"%s"

#: ../apport/ui.py:1960 ../apport/ui.py:1972
msgid "Problem already known"
msgstr "이미 알려진 문제"

#: ../apport/ui.py:1962
msgid ""
"This problem was already reported in the bug report displayed in the web "
"browser. Please check if you can add any further information that might be "
"helpful for the developers."
msgstr ""
"이 문제는 웹 브라우저에 표시된 버그 보고서에 이미 보고된 것입니다. 개발자에"
"게 도움이 될 수 있는 추가 정보를 제공할 수 있는지 확인해 주십시오."

#: ../apport/ui.py:1974
msgid "This problem was already reported to developers. Thank you!"
msgstr "이미 문제를 개발자에게 보고했습니다. 감사합니다!"

#: ../bin/apport-cli.py:84
msgid "Press any key to continue..."
msgstr "아무 키나 누르면 계속합니다..."

#: ../bin/apport-cli.py:91
msgid "What would you like to do? Your options are:"
msgstr "다음 옵션 가운데 어떤 것을 실행하시겠습니까?:"

#: ../bin/apport-cli.py:104
#, python-format
msgid "Please choose (%s):"
msgstr "선택해 주십시오 (%s):"

#: ../bin/apport-cli.py:172
#, python-format
msgid "(%i bytes)"
msgstr "(%i 바이트)"

#: ../bin/apport-cli.py:174 ../gtk/apport-gtk.py:169 ../kde/apport-kde.py:406
msgid "(binary data)"
msgstr "(바이너리 데이터)"

#: ../bin/apport-cli.py:209 ../gtk/apport-gtk.py:208 ../gtk/apport-gtk.ui.h:6
#: ../kde/apport-kde.py:199
msgid "Send problem report to the developers?"
msgstr "문제를 개발자에게 보고하시겠습니까?"

#: ../bin/apport-cli.py:211
msgid ""
"After the problem report has been sent, please fill out the form in the\n"
"automatically opened web browser."
msgstr ""
"문제 보고서가 전송된 후에, 자동으로 열리는 웹 브라우저 내의 문항을\n"
"작성해 주시기 바랍니다."

#: ../bin/apport-cli.py:218
#, python-format
msgid "&Send report (%s)"
msgstr "보고서 보내기(&S) - (%s)"

#: ../bin/apport-cli.py:223
msgid "&Examine locally"
msgstr "로컬에서 검사(&E)"

#: ../bin/apport-cli.py:227
msgid "&View report"
msgstr "보고서 보기(&V)"

#: ../bin/apport-cli.py:230
msgid "&Keep report file for sending later or copying to somewhere else"
msgstr "나중에 보내거나 다른 곳에 복사해 두기 위해 보고서 파일 저장(&K)"

#: ../bin/apport-cli.py:235
msgid "Cancel and &ignore future crashes of this program version"
msgstr "이 버전의 프로그램에서 다시 문제가 발생해도 무시하기(&I)"

#: ../bin/apport-cli.py:238 ../bin/apport-cli.py:332 ../bin/apport-cli.py:366
#: ../bin/apport-cli.py:389
msgid "&Cancel"
msgstr "취소(&C)"

#: ../bin/apport-cli.py:273
msgid "Problem report file:"
msgstr "문제 보고서 파일:"

#: ../bin/apport-cli.py:279 ../bin/apport-cli.py:284
msgid "&Confirm"
msgstr "확인(&C)"

#: ../bin/apport-cli.py:283
#, python-format
msgid "Error: %s"
msgstr "오류: %s"

#: ../bin/apport-cli.py:289 ../kde/apport-kde.py:462
msgid "Collecting problem information"
msgstr "문제 정보를 모으는 중"

#: ../bin/apport-cli.py:291
msgid ""
"The collected information can be sent to the developers to improve the\n"
"application. This might take a few minutes."
msgstr ""
"수집한 정보를 개발자에게 보내 프로그램 개선할 수 있습니다.\n"
"이 작업은 수 분이 필요할 수 있습니다."

#: ../bin/apport-cli.py:307 ../gtk/apport-gtk.ui.h:17 ../kde/apport-kde.py:511
msgid "Uploading problem information"
msgstr "문제 정보를 업로드하는 중"

#: ../bin/apport-cli.py:309
msgid ""
"The collected information is being sent to the bug tracking system.\n"
"This might take a few minutes."
msgstr ""
"수집한 정보가 버그 추적 시스템으로 전송되고 있습니다.\n"
"이 과정은 몇 분 정도 걸릴 수 있습니다."

#: ../bin/apport-cli.py:365
msgid "&Done"
msgstr "마침(&D)"

#: ../bin/apport-cli.py:371
msgid "none"
msgstr "없음"

#: ../bin/apport-cli.py:373
#, python-format
msgid "Selected: %s. Multiple choices:"
msgstr "%s. 다중 선택:"

#: ../bin/apport-cli.py:390
msgid "Choices:"
msgstr "선택:"

#: ../bin/apport-cli.py:404
msgid "Path to file (Enter to cancel):"
msgstr "파일 경로(취소하려면 엔터 키를 누르세요):"

#: ../bin/apport-cli.py:410
msgid "File does not exist."
msgstr "파일이 존재하지 않습니다."

#: ../bin/apport-cli.py:412
msgid "This is a directory."
msgstr "이것은 디렉터리입니다."

#: ../bin/apport-cli.py:418
msgid "To continue, you must visit the following URL:"
msgstr "계속하려면, 다음 주소를 방문해주십시오:"

#: ../bin/apport-cli.py:421
msgid ""
"You can launch a browser now, or copy this URL into a browser on another "
"computer."
msgstr ""
"지금 브라우저를 실행하거나 다른 컴퓨터의 브라우저에 이 주소를 복사해주십시오."

#: ../bin/apport-cli.py:427
msgid "Launch a browser now"
msgstr "지금 브라우저 실행하기"

#: ../bin/apport-cli.py:443
msgid "No pending crash reports. Try --help for more information."
msgstr ""
"대기 중인 문제 보고서가 없습니다. --help 옵션을 이용해 더 많은 정보를 확인할 "
"수 있습니다."

#: ../bin/apport-retrace.py:44
msgid "Do not put the new traces into the report, but write them to stdout."
msgstr "새로운 오류 추적 기록을 보고서에 추가하지 않고 STDOUT로 내보냅니다."

#: ../bin/apport-retrace.py:53
msgid ""
"Start an interactive gdb session with the report's core dump (-o ignored; "
"does not rewrite report)"
msgstr ""
"보고서의 코어 덤프로 gdp 세션 시작.(-o 무시, 보고서를 다시 작성하지 않음)"

#: ../bin/apport-retrace.py:62
msgid ""
"Write modified report to given file instead of changing the original report"
msgstr "원본 보고서를 바꾸지 않고, 수정된 보고서를 주어진 파일에 씁니다"

#: ../bin/apport-retrace.py:72
msgid "Remove the core dump from the report after stack trace regeneration"
msgstr "스텍 트레이스를 다시 생성한 후 코어 덤프 제거"

#: ../bin/apport-retrace.py:80
msgid "Override report's CoreFile"
msgstr "보고서의 코어 파일 덮어쓰기"

#: ../bin/apport-retrace.py:86
msgid "Override report's ExecutablePath"
msgstr "보고서의 실행 경로(ExecutablePath) 덮어쓰기"

#: ../bin/apport-retrace.py:92
msgid "Override report's ProcMaps"
msgstr "보고서의 프로세스 맵(ProcMaps) 덮어쓰기"

#: ../bin/apport-retrace.py:98
msgid "Rebuild report's Package information"
msgstr "보고서의 패키지 정보 다시쓰기"

#: ../bin/apport-retrace.py:105
msgid ""
"Build a temporary sandbox and download/install the necessary packages and "
"debug symbols in there; without this option it assumes that the necessary "
"packages and debug symbols are already installed in the system. The argument "
"points to the packaging system configuration base directory; if you specify "
"\"system\", it will use the system configuration files, but will then only "
"be able to retrace crashes that happened on the currently running release."
msgstr ""
"임시 샌드박스를 만들고 필요한 패키지 다운로드를 다운로드한 후  설치하였으며 "
"디버그 심볼을 만들었습니다; 이 옵션이 없는 경우 필요한 패키지와 디버그 심볼"
"을 시스템에 이미 설치한 것으로 간주합니다. 인수가 패키징 시스템 설정 베이스 "
"시스템을 가리킵니다; \"시스템\"을 지정하면 시스템 설정 파일을 사용하지만 그"
"런 후 현재 실행 중인 배포판에서 발생한 충돌 만을 다시 추적할 수 있습니다."

#: ../bin/apport-retrace.py:119
msgid ""
"Build another temporary sandbox for installing gdb and its dependencies "
"using the same release as the report rather than whatever version of gdb you "
"have installed."
msgstr ""
"설치 한 gdb의 버전이 아닌 보고서와 동일한 릴리스를 사용하여 gdb 및 그 종속성"
"을 설치하기 위한 또 다른 임시 샌드 박스를 빌드하십시오."

#: ../bin/apport-retrace.py:129
msgid "Report download/install progress when installing packages into sandbox"
msgstr "샌드박스에 패키지를 설치할 때 다운로드 및 설치 과정을 보고합니다."

#: ../bin/apport-retrace.py:136
msgid "Prepend timestamps to log messages, for batch operation"
msgstr "배치 작업을 위해 기록 메시지에 시간 기록을 추가합니다."

#: ../bin/apport-retrace.py:142
msgid ""
"Create and use third-party repositories from origins specified in reports"
msgstr "보고서에 지정한 원본 위치의 외부 저장소를 만들고 사용합니다."

#: ../bin/apport-retrace.py:150
msgid "Cache directory for packages downloaded in the sandbox"
msgstr "샌드박스에 다운로드한 패키지를 보관할 캐시 디렉터리"

#: ../bin/apport-retrace.py:156
msgid ""
"Directory for unpacked packages. Future runs will assume that any already "
"downloaded package is also extracted to this sandbox."
msgstr ""
"압축을 푼 패키지를 보관할 디렉터리입니다. 다음 실행 시점부터는 이미 다운로드"
"한 패키지를 이 샌드박스에 압축을 해제한 것으로 추정합니다."

#: ../bin/apport-retrace.py:167 ../bin/apport-valgrind.py:97
msgid ""
"Install an extra package into the sandbox (can be specified multiple times)"
msgstr "추가 패키지를 샌드박스에 설치(여러 항목을 지정할 수 있음)"

#: ../bin/apport-retrace.py:174
msgid ""
"Path to a file with the crash database authentication information. This is "
"used when specifying a crash ID to upload the retraced stack traces (only if "
"neither -g, -o, nor -s are specified)"
msgstr ""
"충돌 데이터베이스 인증 정보를 가진 파일 경로. 스텍 트레이스를 업로드하기 위"
"한 충돌 ID를 설정할 때 사용합니다.(-g,-o나 -s가 설정되지 않은 경우)"

#: ../bin/apport-retrace.py:184
msgid ""
"Display retraced stack traces and ask for confirmation before sending them "
"to the crash database."
msgstr ""
"스텍 트레이스를 보여주고 오작동(crash) 데이터베이스로 보내기 전에 확인하기."

#: ../bin/apport-retrace.py:192
msgid "Path to the duplicate sqlite database (default: no duplicate checking)"
msgstr "중복 sqlite 데이터베이스 경로( 기본: 중복 확인 없음)"

#: ../bin/apport-retrace.py:200
msgid "Do not add StacktraceSource to the report."
msgstr "StacktraceSource를 보고서에 추가하지 않기."

#: ../bin/apport-retrace.py:213
msgid "You cannot use -C without -S. Stopping."
msgstr "-S 옵션을 없이 -C 옵션을 사용할 수 없습니다. 멈춥니다."

#. translators: don't translate y/n,
#. apport currently only checks for "y"
#: ../bin/apport-retrace.py:243
msgid "OK to send these as attachments? [y/n]"
msgstr "첨부 파일로 보내려면, 확인을 누르시오"

#: ../bin/apport-unpack.py:29
#, python-format
msgid "%(prog)s <report> <target directory>"
msgstr ""

#: ../bin/apport-unpack.py:31
msgid "Report file to unpack"
msgstr ""

#: ../bin/apport-unpack.py:33
msgid "directory to unpack report to"
msgstr ""

#: ../bin/apport-unpack.py:59
msgid "Destination directory exists and is not empty."
msgstr "디렉터리는 존재하지만, 파일이 없습니다."

#: ../bin/apport-valgrind.py:37
msgid "See man page for details."
msgstr "자세한 내용은 맨 페이지를 확인하십시오."

#: ../bin/apport-valgrind.py:46
msgid "specify the log file name produced by valgrind"
msgstr "밸그린드가 만든 로그 파일의 이름을 지정"

#: ../bin/apport-valgrind.py:52
msgid ""
"reuse a previously created sandbox dir (SDIR) or, if it does not exist, "
"create it"
msgstr ""
"이전에 만든 샌드박스 디렉터리(SDIR)를 다시 사용하거나 - 존재하지 않는 경우 - "
"만듭니다."

#: ../bin/apport-valgrind.py:60
msgid ""
"do  not  create  or reuse a sandbox directory for additional debug symbols "
"but rely only on installed debug symbols."
msgstr ""
"추가 디버그 심볼이 사용할 샌드박스 디렉터리를 다시 사용하거나 만들지 않고 설"
"치한 디버그 심볼에만 의존합니다."

#: ../bin/apport-valgrind.py:69
msgid ""
"reuse a previously created cache dir (CDIR) or, if it does not exist, create "
"it"
msgstr ""
"이전에 만든 캐시 디렉터리(SDIR)를 다시 사용하거나 - 존재하지 않는 경우 - 만듭"
"니다."

#: ../bin/apport-valgrind.py:78
msgid "report download/install progress when installing packages into sandbox"
msgstr "패키지를 샌드박스에 설치할 때 다운로드/설치 진행 상태를 보고합니다."

#: ../bin/apport-valgrind.py:86
msgid ""
"the executable that is run under valgrind's memcheck tool for memory leak "
"detection"
msgstr ""

#: ../bin/apport-valgrind.py:129
#, python-format
msgid "Error: %s is not an executable. Stopping."
msgstr "오류: %s은(는) 실행할 수 없습니다. 멈춥니다."

#: ../data/apportcheckresume.py:68
msgid ""
"This occurred during a previous suspend, and prevented the system from "
"resuming properly."
msgstr ""
"이전에 절전을 했을 때 발생하였고 시스템이 정상적으로 계속 진행하는 것을 차단"
"하고 있습니다."

#: ../data/apportcheckresume.py:73
msgid ""
"This occurred during a previous hibernation, and prevented the system from "
"resuming properly."
msgstr ""
"이전에 최대 절전을 했을 때 발생하였고 시스템이 정상적으로 계속 진행하는 것을 "
"차단하고 있습니다."

#: ../data/apportcheckresume.py:81
msgid ""
"The resume processing hung very near the end and will have appeared to have "
"completed normally."
msgstr ""
"재개 과정의 처리가 거의 완료된 후 중단되어, 정상적으로 완료된 것처럼 보였을 "
"것입니다."

#: ../data/kernel_oops.py:30
msgid "Your system might become unstable now and might need to be restarted."
msgstr "현재 시스템이 불안정하여 컴퓨터를 다시 시작해야 합니다."

#: ../gtk/apport-gtk.desktop.in.h:1 ../kde/apport-kde-mime.desktop.in.h:1
#: ../kde/apport-kde.desktop.in.h:1
msgid "Report a problem..."
msgstr "문제 보고..."

#: ../gtk/apport-gtk.desktop.in.h:2 ../kde/apport-kde-mime.desktop.in.h:2
#: ../kde/apport-kde.desktop.in.h:2
msgid "Report a malfunction to the developers"
msgstr "개발자에게 버그를 보고합니다"

#: ../gtk/apport-gtk.py:185
#, python-format
msgid "Sorry, the application %s has stopped unexpectedly."
msgstr "프로그램 %s이(가) 예상치 않게 끝났습니다."

#: ../gtk/apport-gtk.py:188
#, python-format
msgid "Sorry, %s has closed unexpectedly."
msgstr "프로그램 %s이(가) 예상치 않게 끝났습니다."

#: ../gtk/apport-gtk.py:193 ../kde/apport-kde.py:214 ../kde/apport-kde.py:266
#, python-format
msgid "Sorry, %s has experienced an internal error."
msgstr "죄송합니다. %s에 내부 오류가 발생했습니다."

#: ../gtk/apport-gtk.py:216 ../gtk/apport-gtk.ui.h:14 ../kde/apport-kde.py:207
msgid "Send"
msgstr "보내기"

#: ../gtk/apport-gtk.py:234 ../gtk/apport-gtk.py:666 ../gtk/apport-gtk.ui.h:11
#: ../kde/apport-kde.py:330
msgid "Show Details"
msgstr "자세한 내용 보이기"

#: ../gtk/apport-gtk.py:247 ../kde/apport-kde.py:258 ../kde/apport-kde.py:276
msgid "Continue"
msgstr "계속"

#: ../gtk/apport-gtk.py:271
#, python-format
msgid "The application %s has stopped responding."
msgstr "프로그램 %s이(가) 응답을 하지 않습니다."

#: ../gtk/apport-gtk.py:275
#, python-format
msgid "The program \"%s\" has stopped responding."
msgstr "프로그램 \"%s\"이(가) 응답을 하지 않습니다."

#: ../gtk/apport-gtk.py:292 ../kde/apport-kde.py:222
#, python-format
msgid "Package: %s"
msgstr "패키지: %s"

#: ../gtk/apport-gtk.py:298 ../kde/apport-kde.py:229
msgid "Sorry, a problem occurred while installing software."
msgstr "죄송합니다. 소프트웨어 설치 중 오류가 발생했습니다."

#: ../gtk/apport-gtk.py:310 ../gtk/apport-gtk.py:334 ../kde/apport-kde.py:238
#, python-format
msgid "The application %s has experienced an internal error."
msgstr "프로그램 %s에 내부 오류가 발생했습니다."

#: ../gtk/apport-gtk.py:316 ../kde/apport-kde.py:245
#, python-format
msgid "The application %s has closed unexpectedly."
msgstr "프로그램 %s이(가) 예상치 않게 끝났습니다."

#: ../gtk/apport-gtk.py:347 ../gtk/apport-gtk.ui.h:7 ../kde/apport-kde.py:271
msgid "If you notice further problems, try restarting the computer."
msgstr "더욱 심각한 문제가 발생한다면 컴퓨터를 다시 시작하십시오."

#: ../gtk/apport-gtk.py:352 ../kde/apport-kde.py:278
msgid "Ignore future problems of this type"
msgstr "앞으로 이런 형식의 문제를 무시합니다."

#: ../gtk/apport-gtk.py:670 ../kde/apport-kde.py:327
msgid "Hide Details"
msgstr "자세한 내용 숨기기"

#: ../gtk/apport-gtk.ui.h:1 ../kde/apport-kde.py:552 ../kde/apport-kde.py:595
msgid "Apport"
msgstr "Apport"

#: ../gtk/apport-gtk.ui.h:2
msgid "Cancel"
msgstr "취소"

#: ../gtk/apport-gtk.ui.h:3
msgid "OK"
msgstr "확인"

#: ../gtk/apport-gtk.ui.h:4
msgid "Crash report"
msgstr "충돌 보고서"

#: ../gtk/apport-gtk.ui.h:5
msgid "<big><b>Sorry, an internal error happened.</b></big>"
msgstr "<big><b>죄송합니다. 내부 오류가 발생했습니다..</b></big>"

#: ../gtk/apport-gtk.ui.h:8
msgid "Remember this in future"
msgstr "이후에도 기억하기"

#: ../gtk/apport-gtk.ui.h:9
msgid "Ignore future problems of this program version"
msgstr "이후 이 버전의 프로그램에서  발생하는 문제를 무시합니다."

#: ../gtk/apport-gtk.ui.h:10
msgid "Relaunch this application"
msgstr "응용 프로그램 다시 시작"

#: ../gtk/apport-gtk.ui.h:12
msgid "_Examine locally"
msgstr "로컬에서 검사(_E)"

#: ../gtk/apport-gtk.ui.h:13
msgid "Don't send"
msgstr "보내지 않기"

#: ../gtk/apport-gtk.ui.h:15
msgid "<big><b>Collecting problem information</b></big>"
msgstr "<big><b>문제 정보를 수집하는 중</b></big>"

#: ../gtk/apport-gtk.ui.h:16
msgid ""
"Information is being collected that may help the developers fix the problem "
"you report."
msgstr ""
"개발자가 여러분이 보낸 보고서를 통해 문제를 해결할 수 있도록 정보를 수집하고 "
"있습니다."

#: ../gtk/apport-gtk.ui.h:18
msgid "<big><b>Uploading problem information</b></big>"
msgstr "<big><b>문제 정보를 업로드하는 중</b></big>"

#: ../gtk/apport-gtk.ui.h:19 ../kde/apport-kde.py:513
msgid ""
"The collected information is being sent to the bug tracking system. This "
"might take a few minutes."
msgstr ""
"수집한 정보를 버그 추적 시스템으로 전송하고 있습니다. 이 작업은 수 분이 필요"
"할 수 있습니다."

#: ../kde/apport-kde-mimelnk.desktop.in.h:1
msgid "Apport crash file"
msgstr "충돌 파일 보고"

#: ../kde/apport-kde.py:261
msgid "Leave Closed"
msgstr "닫은 채로 두기"

#: ../kde/apport-kde.py:262 ../kde/apport-kde.py:432
msgid "Relaunch"
msgstr "다시 시작"

#: ../kde/apport-kde.py:355
msgid "Username:"
msgstr "사용자 이름:"

#: ../kde/apport-kde.py:356
msgid "Password:"
msgstr "암호:"

#: ../kde/apport-kde.py:461
msgid "Collecting Problem Information"
msgstr "문제 정보를 모으는 중"

#: ../kde/apport-kde.py:464
msgid ""
"The collected information can be sent to the developers to improve the "
"application. This might take a few minutes."
msgstr ""
"수집한 정보는 프로그램을 개선하기 위해 개발자에게 보낼 수 있습니다. 이 작업"
"은 수 분이 필요할 수 있습니다."

#: ../kde/apport-kde.py:510
msgid "Uploading Problem Information"
msgstr "문제 정보를 업데이트 하는 중"
